### DISCUSSION TITLE
Is session/token persistent between deploys?

### DISCUSSION THREAD

**Discussion by jcubic** [gladly-team/next-firebase-auth#309] [Q&A] [answered]
First quick intro, I'm very new to NextJS, Firebase, Serverless, and all other related technologies. I'm still learning so I hope my question is not stupid.

When I was reading about serverless on Vercel, everything is cleared when a new version is deployed to the server (on git push).
So my question is this if I push a new version to the server, will the user keep logged in? Or will they all be logout and need to log in again?

**Comment by kmjennison** [comment, marked as answer]
It will keep them logged in. Their authentication info is stored on their device, in browser storage, so it's not affected by simply redeploying the app.
